cellpack/tests/test_recipe_loader.py

```diff
@@ -67,3 +67,40 @@ def test_resolve_objects():
     # assert resolved_objects["sphere_50"]["radius"] == 50
     # TODO: add granular testing for individual objects
     assert resolved_objects == expected_result
+
+
+@pytest.mark.parametrize(
+    "input_recipe_data, expected_result",
+    [
+        (
+            {},
+            "1.0",
+        ),
+        (
+            {"format_version": "2.0.33"},
+            "2.0",
+        ),
+        (
+            {"format_version": "2.33.0"},
+            "2.33",
+        ),
+        (
+            {"format_version": "2.0"},
+            "2.0",
+        ),
+        (
+            {"format_version": "2.105"},
+            "2.105",
+        ),
+        (
+            {"format_version": "2"},
+            "2.0",
+        ),
+        (
+            {"format_version": "1"},
+            "1.0",
+        ),
```

Comment:
These cover the cases we expect pretty well. Do we want to add some freaky cases like `"a;slfhaofhweihfkd@@@"` or `"(((((------.828282828.18181818"`? May not be necessary but you never know what people come up with.